.github/workflows/process_data.py

```
@@ -10,101 +11,94 @@
 KEY_IDENTITY = os.getenv("KEY_IDENTITY")
 KEY_CREDENTIAL = os.getenv("KEY_CREDENTIAL")
 ITEM_SET_ID = os.getenv("ITEM_SET_ID")
+if not ITEM_SET_ID:
+    raise ValueError("ITEM_SET_ID environment variable must be set")
+try:
+    ITEM_SET_ID = int(ITEM_SET_ID)
+except ValueError:
+    raise ValueError("ITEM_SET_ID must be a valid integer")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)